portfolio/src/main/webapp/script.js

```diff
@@ -41,20 +41,24 @@ function normal(image) {
 }
 
 function loadTasks() {
-  fetch('/data').then(response => response.json()).then((tasks) => {
-    const taskListElement = document.getElementById('task-list');
+  const parameter = document.getElementById("parameter").value;
+  console.log(document.getElementById("parameter").value);
+  fetch("/data?parameterValue=" + parameter).then(response => response.json()).then((tasks) => {
+    const taskListElement = document.getElementById("task-list");
     tasks.forEach((task) => {
+      const linebreak = document.createElement("br");
```

Comment:
Since this is the same each time, you can declare it outside of the for loop (like where you declare taskListElement).